src/core/util/generators/uiSchema.ts

```diff
@@ -7,13 +7,27 @@
  */
 import { ControlElement, Layout } from '@jsonforms/core';
 
-export const createControl = (scope: string): ControlElement => {
+import { SchemaElement } from '../../model';
+
+export const createControl = (
+  scope: string,
+  schema: SchemaElement
+): ControlElement => {
   return {
     type: 'Control',
     scope: scope,
+    options: getOptionsForControlWithScope(scope, schema),
   };
 };
 
+const getOptionsForControlWithScope = (
+  scope: string,
+  schema: SchemaElement
+): { [key: string]: any } => {
+  //TODO generate options based on schema element
+  return { multi: false };
```

Comment:
I think we need a way to set a schema and uischema for the properties view, maybe it makes sense to retrieve them in the properties view instead of passing data using options